OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.h

```diff
@@ -140,37 +130,37 @@ namespace OpenSim {
     @note The ExternalLoads used by this goal is separate from the model. Using
     this goal implies that the model contains compliant contact forces, so
     adding ExternalLoads to the model would be redundant. This class uses the
-    ExternalLoads *only* for computing the force error, not for applying forces
-    to the model.
+    ExternalLoads *only* for computing the contact impulse error, not for 
+    applying forces to the model.
     ### Scale factors
     Add a MocoParameter to the problem that will scale the tracking reference
     data associated with a contact force group. Scale factors are applied
     to the tracking error calculations based on the following equation:
          error = modelValue - scaleFactor * referenceValue
-    In other words, the scale factor is applied when computing the tracking
-    error for each contact force group, not to the reference data directly.
-    You must specify both the external force name associated with the contact
-    force group and the index corresponding to the direction (i.e., X = 0,
-    Y = 1, Z = 2) of the scaled force value. The direction is applied in
-    whatever frame the reference data is expressed in based on the provided
-    ExternalLoads in each contact group.
+    In other words, the scale factor is applied when computing the contact
+    impulse tracking error for each contact force group, not to the reference 
+    data directly. You must specify both the external force name associated 
+    with the contact force group and the index corresponding to the direction
+    (i.e., X = 0, Y = 1, Z = 2) of the scaled force value. The direction is 
+    applied in whatever frame the reference data is expressed in based on the 
+    provided ExternalLoads in each contact group.
     Adding a scale factor to a MocoImpulseTrackingGoal.
     @code
-    auto* markerTrackingGoal = problem.addGoal<MocoContactTrackingGoal>();
+    auto* contactImpulseTrackingGoal = problem.addGoal<MocoContactImpulseTrackingGoal>();
     ...
-    markerTrackingGoal->addScaleFactor(
+    contactImpulseTrackingGoal->addScaleFactor(
             'RightGRF_vertical_scale_factor', 'Right_GRF', 1, {0.5, 2.0});
     @endcode
     @ingroup mocogoal */
-    class OSIMMOCOIMPULSETRACKINGGOAL_API MocoImpulseTrackingGoal : public MocoGoal {
-        OpenSim_DECLARE_CONCRETE_OBJECT(MocoImpulseTrackingGoal, MocoGoal);
+    class OSIMMOCOCONTACTIMPULSETRACKINGGOAL_API MocoContactImpulseTrackingGoal : public MocoGoal {
```

Comment:
Since we'll be adding this directly to the OpenSim API, we'll use a different macro here. Replace `OSIMMOCOCONTACTIMPULSETRACKINGGOAL_API` with `OSIMMOCO_API`.